docs/custom-workload.md

```diff
@@ -0,0 +1,115 @@
+# Deploying a custom workload
+
+## Example workload
+
+A typical windows deployment would look like this.
```

Comment:
This is actually an atypical  deployment. It is usually recommended not use the security contexts we have defined here. We only do it because we need to run the http server. This is something we need to fix in the OpenShift docs also.

Reply:
:eyes: what security contexts should we be using instead? Should we get rid of it entirely?

Reply:
Just don't call it a typcial workload but rather an example work load. We can address the security context bits separately.